src/pip/_internal/utils/subprocess.py

```diff
@@ -191,8 +186,7 @@ def call_subprocess(
         raise
     all_output = []
     while True:
-        # The "line" value is a unicode string in Python 2.
-        line = console_to_str(proc.stdout.readline())
+        line = proc.stdout.readline()
```

Comment:
Probably worth a `# type: str` and maybe a comment so future readers know the type of `line` is `str` and why (because we specified `errors`).

Reply:
Makes sense to me. Added in the latest revision.